helper.go

```diff
@@ -0,0 +1,31 @@
+package email
+
+import "io"
+
+// ChunkedReader provides reading by specified portion size.
+// Only last chunk can be lesser or zero-size same time with EOF or other error
+type ChunkedReader struct {
+	r        io.Reader
+	chunkLen int
+}
+
+func (cr *ChunkedReader) Read(b []byte) (int, error) {
+	accumulatedBytes := 0
+	if len(b) < cr.chunkLen {
+		return 0, io.ErrShortBuffer
+	}
+	var err error
+	var n int
+	for accumulatedBytes < cr.chunkLen && err == nil {
+		n, err = cr.r.Read(b[accumulatedBytes:])
+		accumulatedBytes += n
+	}
```

Comment:
In this case I disagree. I think, that ReadFull is not suitable here. It works fine, when we estimate certain length of content, but when we don't know forehead the len of content, it can produce ErrUnexpectedEOF in normal data flow. 
So I specially used chunked reader to obtain data splitted on same chunks except the only one last chunk, which can be shorter. (I didn't found a standard reader with same behavior)

Reply:
Yes, if less then chunkLen bytes of data remain, we would get back `ErrUnexpectedEOF` and have to handle that as success.
This is not ideal but we could get rid of the whole loop and the code would get more simple.
I think think that trade-off is worth it.

I'm also fine with keeping the current code, your choice. :-)